smarts/core/agent_interface.py

```diff
@@ -97,6 +97,11 @@ class NeighborhoodVehicles:
     radius: float = None
 
 
+@dataclass
+class Accelerometer:
+    ...
```

Comment:
This is intentionally left blank, or you plan to add fields?

Reply:
Also ellipses are fine and cute, but project has been using `pass` until now.

Reply:
It is intentionally blank.